### PR TITLE
Ld disc stacker enhanced

### PR DESCRIPTION
**Pull request by happycube**

### CODE REVIEW

File: tools/ld-disc-stacker/main.cpp [hunk 1]
```diff
@@ -86,14 +86,27 @@ int main(int argc, char *argv[])
...
                                         QCoreApplication::translate("main", "number"));
     parser.addOption(threadsOption);
+	
+    // Option to select the number of threads (-t)
+    QCommandLineOption modeOption(QStringList() << "m" << "mode",
+                                        QCoreApplication::translate(
+                                         "main", "Specify the stacking mode to use (default is 2) 0 = mean / 1 = median / 2 = smart"),
+    parser.addOption(modeOption);
+	
+	// Option to select the number of threads (-t)
```
Comment (ifb) [inline tools/ld-disc-stacker/main.cpp:97]: Same.

File: tools/ld-disc-stacker/stacker.cpp [hunk 2]
```diff
@@ -40,6 +40,8 @@ void Stacker::run()
     QVector<SourceVideo::Data> secondSourceField;
     QVector<LdDecodeMetaData::Field> firstFieldMetadata;
     QVector<LdDecodeMetaData::Field> secondFieldMetadata;
+	qint32 mode;
```
Comment (ifb) [inline tools/ld-disc-stacker/stacker.cpp:43]: Indentation (pretty sure a tab snuck in instead of spaces).

File: tools/ld-disc-stacker/main.cpp [hunk 3]
```diff
@@ -124,6 +137,28 @@ int main(int argc, char *argv[])
     bool noDiffDod = parser.isSet(noDiffDodOption);
     bool noMap = parser.isSet(noMapOption);
     bool passThrough = parser.isSet(passthroughOption);
+	
+    // Get the arguments from the parser
+	qint32 mode = 2;
```
Comment (ifb) [inline tools/ld-disc-stacker/main.cpp:143]: Indentation on these if() blocks is wrong.

File: tools/ld-disc-stacker/main.cpp [hunk 4]
```diff
@@ -124,6 +137,28 @@ int main(int argc, char *argv[])
...
+        mode = parser.value(modeOption).toInt();
+
+        if (mode > 2 || mode < 0) {
+            qInfo() << "Specified mode (" << mode << ") is unknown using 2 (smart) instead";
+            mode = 2;
+        }
+    }
+	
+    // Get the arguments from the parser
+	qint32 smartTreshold = (15*256);
```
Comment (ifb) [inline tools/ld-disc-stacker/main.cpp:154]: Same (indentation)

File: tools/ld-disc-stacker/main.cpp (hunk 3 above)
Comment (happycube) [reply tools/ld-disc-stacker/main.cpp]: Still wrong - please convert the tabs here to 4 spaces.

File: tools/ld-disc-stacker/main.cpp (hunk 1 above)
Comment (happycube) [reply tools/ld-disc-stacker/main.cpp]: Looks fine now.

**Pull request merged**